### DISCUSSION TITLE
Multiple spotlights?

### DISCUSSION THREAD

**Discussion by dankolesnikov** [mantinedev/mantine#1076] [Q&A] [answered]
is there a way to have multiple instances of spotlight with the ability to open a particular spotlight using a hook similar to modals?

Example:

```
const spotlights = useSpotlight()
if(isOpened) {
spotlights.openSpotlight({id: 123});
}
```

**Comment by rtivital** [comment, marked as answer]
You can have as many spotlights as you need, `useSpotlight` hook will take the value from the nearest context

**Comment by sandcastle** [comment]
that's a bit of shame, i would love to be able to have multiple spotlights that are keyed with different shortcut combos

**Reply by rtivital** [reply]
You can use as many spotlighs as you need, if you bind them to different keyboard shortcuts.

**Reply by sandcastle** [reply]
that's great, i didn't realise that was possible
